dcicutils/beanstalk_utils.py

```diff
@@ -32,37 +44,24 @@
 # use a 'try' expression to sort things out and call the safe function 'use_input' to avoid confusion.
 # But PyCharm found that 'try' expression confusing, so now that we are Python 3 only, we're phasing that
 # out. For a time, we'll retain the transitional naming, though, along with an affirmative error check, so
-# we don't open any security holes. We can remove this naming and check once we're we're only using Python 3.
+# we don't open any security holes.
+# TODO: We can remove this naming and check once we're we're only using Python 3.
 # -kmp 27-Mar-2020
 
-_python_major_version = sys.version_info[0]
-if _python_major_version < 3:
-    raise EnvironmentError("The 'dcicutils.beanstalk_utils' package only works in Python 3.")
 use_input = input  # In Python 3, this does 'safe' input reading.
 
+whodaman = compute_ff_prd_env  # This naming is obsolete but retained for compatibility.
```

Comment:
We don't, so if you want to remove it feel free.

Reply:
I'll remove whodaman on the next major version, which I'm going to do as a separate PR since this one is already checked as-is. At the same time, I'll want to tighten up which python versions we're using, and I don't want to do that here.